spot-setup/hdfs_setup.sh

```diff
@@ -1,35 +1,104 @@
 #!/bin/bash
 
+if [ $# -eq 1 ]
+then
+	SUPERGROUP=$1
+else
+	SUPERGROUP="supergroup"
+fi
+  
 DSOURCES=('flow' 'dns' 'proxy')
 DFOLDERS=('binary' 'hive' 'stage')
-source /etc/spot.conf
 
+if [ ! -e "/etc/spot.conf" ]
+then
+	echo "Please, copy spot.conf under /etc/spot.conf and edit it before running this script"
+	exit 1
+fi
+
+if ! groups | grep -c "\b${SUPERGROUP}\b" &>/dev/null
+then
+	echo "Current user doesn't belong to hdfs supergroup '${SUPERGROUP}'"
+	echo "If you've another group configured, please run this script as $0 supergroupname"
+	exit 2
```

Comment:
@kpeiruza I think i would feel more comfortable with more traditional exit codes, If you remove exit 2 then you've have exit 1,3,4 and so on.
Unless someone has an opposing point of view or new suggestion can you work up the changes? Then QA can validate and we can merge